src/genie/libs/parser/iosxr/show_dampening.py

```diff
@@ -0,0 +1,257 @@
+""" show_dampening.py
+    supports commands:
+        * show im dampening
+        * show im dampening {interface}
+"""
+
+# Python
+import re
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Optional
+
+# parser utils
+from genie.libs.parser.utils.common import Common
+
+# =============================================
+# Parser for 'show im dampening'
+# =============================================
+
+class ShowImDampeningSchema(MetaParser):
+    schema = {
+                'interface': {
```

Comment:
Please fix the indentation for this schema

Reply:
Fixed